pkg/plugins/ca/builtin/config/builtin_ca_config.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.23.0
+// 	protoc-gen-go v1.25.0
```

Comment:
upgrade go and do `make dev/tools`

Reply:
Thanks! That was extremely helpful, I had the latest Go version, but didn't call `make dev/tools`